<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>objectForm</class>
 <widget class="QDialog" name="objectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>objectForm</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Rotate Z</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="rotSliderY">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="pageStep">
       <number>36</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="rotSliderZ">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="pageStep">
       <number>36</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="rotSliderX">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="pageStep">
       <number>36</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Rotate Y</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rotate X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="rotValueX">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="rotValueY">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="rotValueZ">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>objectForm</receiver>
   <slot>acceptCB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>objectForm</receiver>
   <slot>rejectCB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotSliderX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>objectForm</receiver>
   <slot>rotSliderXChangedCB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotSliderY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>objectForm</receiver>
   <slot>rotSliderYChangedCB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotSliderZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>objectForm</receiver>
   <slot>rotSliderZChangedCB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rotSliderXChangedCB(int)</slot>
  <slot>rotSliderYChangedCB(int)</slot>
  <slot>rotSliderZChangedCB(int)</slot>
  <slot>acceptCB()</slot>
  <slot>rejectCB()</slot>
 </slots>
</ui>
